tests/test_app/TestApp/Policy/ArticlesTablePolicy.php

```diff
@@ -14,4 +15,18 @@ public function canModify(IdentityInterface $identity)
     {
         return $identity['can_edit'];
     }
+
+    public function scopeEdit(IdentityInterface $user, QueryInterface $query)
+    {
+        return $query->where([
+            'user_id' => $user['id']
+        ]);
+    }
+
+    public function scopeModify(IdentityInterface $user, QueryInterface $query)
+    {
+        return $query->where([
+            'user_ID' => $user['id']
```

Comment:
Should this be `user_id`? Perhaps this should add a different condition as this looks like a typo.